build.sbt

```diff
@@ -52,6 +54,17 @@ lazy val core = myCrossProject("core")
       Dependencies.scalaTest % Test
     ),
     assembly / test := {},
+    assemblyMergeStrategy in assembly := {
+      val nativeSuffix = "\\.(?:dll|jnilib|so)$".r
+
+      {
+        case PathList(ps @ _*) if nativeSuffix.findFirstMatchIn(ps.last).isDefined =>
+          MergeStrategy.first
+        case otherwise =>
+          val defaultStrategy = (assemblyMergeStrategy in assembly).value
+          defaultStrategy(otherwise)
+      }
+    },
```

Comment:
Just to document why this is necessary, here are the errors we get without this change:
```
sbt:scala-steward> core/assembly
[info] Updating core...
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[error] 5 errors were encountered during merge
[error] java.lang.RuntimeException: deduplicate: different file contents found in the following:
[error] /home/frank/.ivy2/cache/org.fusesource.jansi/jansi/jars/jansi-1.18.jar:META-INF/native/linux32/libjansi.so
[error] /home/frank/.ivy2/cache/org.scala-lang/scala-compiler/jars/scala-compiler-2.12.8.jar:META-INF/native/linux32/libjansi.so
[error] deduplicate: different file contents found in the following:
[error] /home/frank/.ivy2/cache/org.fusesource.jansi/jansi/jars/jansi-1.18.jar:META-INF/native/linux64/libjansi.so
[error] /home/frank/.ivy2/cache/org.scala-lang/scala-compiler/jars/scala-compiler-2.12.8.jar:META-INF/native/linux64/libjansi.so
[error] deduplicate: different file contents found in the following:
[error] /home/frank/.ivy2/cache/org.fusesource.jansi/jansi/jars/jansi-1.18.jar:META-INF/native/osx/libjansi.jnilib
[error] /home/frank/.ivy2/cache/org.scala-lang/scala-compiler/jars/scala-compiler-2.12.8.jar:META-INF/native/osx/libjansi.jnilib
[error] deduplicate: different file contents found in the following:
[error] /home/frank/.ivy2/cache/org.fusesource.jansi/jansi/jars/jansi-1.18.jar:META-INF/native/windows32/jansi.dll
[error] /home/frank/.ivy2/cache/org.scala-lang/scala-compiler/jars/scala-compiler-2.12.8.jar:META-INF/native/windows32/jansi.dll
[error] deduplicate: different file contents found in the following:
[error] /home/frank/.ivy2/cache/org.fusesource.jansi/jansi/jars/jansi-1.18.jar:META-INF/native/windows64/jansi.dll
[error] /home/frank/.ivy2/cache/org.scala-lang/scala-compiler/jars/scala-compiler-2.12.8.jar:META-INF/native/windows64/jansi.dll
```